lib/app_files/category_i_files/book_categories.dart

```diff
@@ -38,14 +59,14 @@ class _BookCategories extends State<BookCategories> {
             ListView.builder(
                 physics: NeverScrollableScrollPhysics(),
                 shrinkWrap: true,
-                itemCount: 6,
+                itemCount: categories.length,
                 itemBuilder: ((context, index) {
                   return Container(
                     margin: EdgeInsets.all(Dimentions.height10),
                     child: Row(
                       mainAxisAlignment: MainAxisAlignment.center,
                       children: [
-                        categoryCard(index),
+                        categoryCard(index,categories[index]),
```

Comment:
why you pass the index value here. because you don't use it? isn't it enough of having categories[index]? @ahamedafri

Reply:
othe than this index problem overall code is better